dubbo-remoting/dubbo-remoting-zookeeper/src/main/java/org/apache/dubbo/remoting/zookeeper/curator/CuratorZookeeperClient.java

```diff
@@ -156,7 +161,7 @@ public boolean isConnected() {
     public String doGetContent(String path) {
         try {
             byte[] dataBytes = client.getData().forPath(path);
-            if(dataBytes == null || dataBytes.length == 0){
+            if (dataBytes == null || dataBytes.length == 0) {
                 return null;
             }
             return new String(dataBytes, charset);

```

Comment:
can be rewritten as `return (dataBytes == null || dataBytes.length == 0) ? null : new String(dataBytes, charset);`

Reply:
It have been modified